Signal/src/view controllers/InboxTableViewCell.m

```diff
@@ -50,9 +50,6 @@ -(void)configureWithThread:(TSThread*)thread {
         if([_nameLabel.text length]==0) {
             _nameLabel.text = @"New Group";
         }
-        if(_contactPictureView.image!=nil) {
```

Comment:
we won't merge this PR..

On Fri, Jan 30, 2015 at 5:27 PM, Frederic Jacobs <notifications@github.com>
wrote:

> In Signal/src/view controllers/InboxTableViewCell.m
> <https://github.com/WhisperSystems/Signal-iOS/pull/541#discussion_r23886062>
> :
>
> > @@ -50,9 +50,6 @@ -(void)configureWithThread:(TSThread*)thread {
> >          if([_nameLabel.text length]==0) {
> >              _nameLabel.text = @"New Group";
> >          }
> > -        if(_contactPictureView.image!=nil) {
>
> Is this intended behaviour? @abolishme <https://github.com/abolishme>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WhisperSystems/Signal-iOS/pull/541/files#r23886062>.
>



-- 
Dr. Corbett Moran
Rockets @ http://www.spacex.com
Physics @ http://www.ICS.uzh.ch  <http://www.ICS.uzh.ch>
Dev @ http://circleof6app.com
Dev @ https://whispersystems.org
www.christinecorbettmoran.com